src/shared/components/input/Input.tsx

```diff
@@ -35,17 +34,17 @@ export const Input = ({
 
             <div className='relative flex w-full items-center'>
                 <input
-                    className={twMerge(
+                    className={
                         cn(
```

Comment:
I completely agree with you on the `twMerge` doesn't do any good with custom classes, I think it is a good signal to us to avoid using custom classes, besides this I'd still have `cn` removed instead of `twMerge`  as it can bring some value and prevent potential bugs, for example, what if we need to set different `px`, `py`, `radius` values for the given input, in this case `cn` won't do any good. What do you think @patricio0312rev? :thinking:

Reply:
I rather wait for a usecase that needs custom padding than considering it at this point. Design should avoid customisation like that as we have a general styleguide for buttons it should follow